src/components/certificates/CertificateHome.tsx

```diff
@@ -18,9 +18,9 @@ class Home extends React.Component {
       <>
         <Hero text={this.props.text} textId="hero" tokenVersion="purple"/>
         <div className="website-hero__buttons">
-          <ScrollToButton id="#signup" text={this.props.text} offset={offset} textId="sign_up"/>
```

Comment:
Could we define a more general class than join slack? Since none off these buttons do join slack?

Reply:
I removed "slack" from the className since the button in the heroes of the main page and the certificates back are both for joining/signing up.